pkg/podrestartmonitor/README.md

```diff
@@ -0,0 +1,62 @@
+# Pod Restart Monitor
+
+*Pod Restart Monitor* is a problem daemon in node problem detector. It monitors the number of times a pod has been
+restarted.
+
+The original use case was for monitoring the `aws-node` DaemonSet in EKS clusters. Multiple restarts of that pod can
+indicate a hard to define, but definite underlying condition on a node that makes it unsuitable for scheduling.
+
+### Plugin Configuration
+
+The plugin checks a very simple condition: how many times any container in a pod has restarted in its lifetime. 
+This condition will not go away unless the pod is deleted.
+
+The defaults are tuned for the aws-node daemon in EKS clusters, but any pod can be monitored:
```

Comment:
Already mentioned, but I think the pod selector should not have a default value.